resources/assets/js/app.js

```diff
@@ -48,6 +48,11 @@ Vue.component(
   require('./components/cookie-law/CookieLaw.vue')
 )
 
+Vue.component(
+  'gzero-youtube',
```

Comment:
I prefer the name to be more descriptive something e.g `youtube-embed` instead `gzero-youtube`.